core/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -196,15 +208,6 @@ public GetField field(String name) {
         return fields.values().iterator();
     }
 
-    static final class Fields {
```

Comment:
I don't know, I think I did it because I wanted to reuse these fields and in general we tend to be removing these. It's a harmless change, not that we have to make it everywhere. I think I will :)